### PR TITLE
feat: add additional colors

### PR DESCRIPTION
**Pull request by nmerget**
## Proposed changes

Add additional colors:
`yellow`, `orange`, `red`, `pink`, `violet`, `blue`,
`cyan`, `turquoise`, `green`

## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: showcases/patternhub/pages/foundations/colors/overview.tsx [hunk 1]
```diff
@@ -3,14 +3,45 @@ import DefaultPage from '../../../components/default-page';
...
+			`${color}-bg-transparent-semi`,
+			`${color}-bg-transparent-full`
+		);
+	}
+
+	return colors;
+};
+
 const ColorOverview = () => {
+	const addColors = getAdditionalColors();
```
Comment (mfranzke) [inline showcases/patternhub/pages/foundations/colors/overview.tsx:34]:
```suggestion
	const additionalColors = getAdditionalColors();
```

File: packages/foundations/scss/colors/_placeholder.scss [hunk 2]
```diff
@@ -1,5 +1,6 @@
 @use "sass:string";
 @use "variables";
+@use "variables.additonal" as add-colors;
```
Comment (mfranzke) [inline packages/foundations/scss/colors/_placeholder.scss:3]:
```suggestion
@use "variables.additonal" as additional-colors;
```

File: packages/foundations/scss/colors/_placeholder.scss [hunk 3]
```diff
@@ -187,3 +188,11 @@
 		}
 	}
 }
+
+@each $variant in variables.$variants {
+	@include generate-color-placeholder($variant);
+}
+
+@each $variant in add-colors.$additional-variants {
```
Comment (mfranzke) [inline packages/foundations/scss/colors/_placeholder.scss:196]:
```suggestion
@each $variant in additional-colors.$additional-variants {
```

File: packages/foundations/scss/colors/_index.scss [hunk 4]
```diff
@@ -1,2 +1,7 @@
 @forward "variables";
+@forward "variables.speaking-colors";
+@forward "variables.palette";
+@forward "variables.additonal";
```
Comment (mfranzke) [inline packages/foundations/scss/colors/_index.scss:4]:
```suggestion
@forward "variables.additional";
```

**Review by mfranzke — APPROVED** [review]

File: packages/foundations/scss/colors/_variables.additonal.scss
[the remark this answers is not in the document]
Comment (nmerget) [reply packages/foundations/scss/colors/_variables.additonal.scss:1]: fixed

**Review by mfranzke — DISMISSED** [review]

File: packages/foundations/scss/colors/_default-palette.scss [hunk 5]
```diff
@@ -102,3 +102,249 @@
...
+	--db-orange-on-light: #000;
+	--db-orange-origin-light: #d17d00;
+	--db-orange-hover-light: #ad6600;
+	--db-orange-pressed-light: #8a5100;
+	--db-orange-on-pressed-dark: #2c1600;
+	--db-orange-on-hover-dark: #210f00;
+	--db-orange-on-dark: #000;
+	--db-orange-origin-dark: #f39200;
+	--db-orange-hover-dark: #ffb782;
+	--db-orange-pressed-dark: #ffdbc8;
```
Comment (bruno-sch) [inline packages/foundations/scss/colors/_default-palette.scss:160]: Why are these "speaking colours" only defined for the colour "brand" and for all **additional colours**, not for the others of the main colour scheme?

**Pull request merged**